src/web/core/src/main/java/org/geoserver/web/data/layergroup/LayerGroupEditPage.java

```diff
@@ -34,38 +72,222 @@ public LayerGroupEditPage(PageParameters parameters) {
-                        if (!(c instanceof AbstractLink && "cancel".equals(c.getId()))) {
-                            c.setEnabled(false);
+    private void postInit() {
+        if (!isAuthenticatedAsAdmin() && !isNew) {
+            // global layer groups only editable by full admin
+            if (getPublishedInfo().getWorkspace() == null) {
+                // disable all form components but cancel
+                setInputEnabled(false);
+
+                info(new StringResourceModel("globalLayerGroupReadOnly", this, null).getString());
+            }
+        }
+    }
+            
+    public class LayerGroupTab extends PublishedEditTabPanel<LayerGroupInfo> {
+
+        private static final long serialVersionUID = 2192005814142588155L;
+
+        public LayerGroupTab(String id) {
+            super(id, myModel);
+            initUI();
+        }
+
+        private EnvelopePanel envelopePanel;       
+        protected RootLayerEntryPanel rootLayerPanel;    
+        
+        private void initUI() {
+
+            final WebMarkupContainer rootLayerPanelContainer = new WebMarkupContainer("rootLayerContainer");
+            rootLayerPanelContainer.setOutputMarkupId(true);
+            add(rootLayerPanelContainer);        
+            
+            rootLayerPanel = new RootLayerEntryPanel("rootLayer", getPublishedInfo().getWorkspace(), myModel);
+            rootLayerPanelContainer.add(rootLayerPanel);
+
+            updateRootLayerPanel(getPublishedInfo().getMode());
+            
+            TextField<String> name = new TextField<String>("name");
+            name.setRequired(true);
+            //JD: don't need this, this is validated at the catalog level
+            //name.add(new GroupNameValidator());
+            add(name);
+
+            final DropDownChoice<LayerGroupInfo.Mode> modeChoice = new DropDownChoice<LayerGroupInfo.Mode>("mode", new LayerGroupModeModel(), new LayerGroupModeChoiceRenderer());
+            modeChoice.setNullValid(false);
+            modeChoice.setRequired(true);
+            modeChoice.add(new OnChangeAjaxBehavior() {
+                private static final long serialVersionUID = 8819356789334465887L;
+
+                @Override
+                protected void onUpdate(AjaxRequestTarget target) {
+                    LayerGroupInfo.Mode mode = modeChoice.getModelObject();
+                    updateRootLayerPanel(mode);
+                    target.addComponent(rootLayerPanelContainer);
+                }
+            });
+            
+            add(modeChoice);
+            
+            add(new TextField<String>("title"));
+            add(new TextArea<String>("abstract"));
+            
+            DropDownChoice<WorkspaceInfo> wsChoice = 
+                    new DropDownChoice<WorkspaceInfo>("workspace", new WorkspacesModel(), 
+                            new WorkspaceChoiceRenderer());
+            wsChoice.setNullValid(true);
+            if (!isAuthenticatedAsAdmin()) {
```

Comment:
Squitinting at the diffs a bit longer (this one has been made way harder than necessary to review) it seems that this code block comes from the old AbstractLayerGroupPage, and the extra behavior in the edit/new pages has been simply lost.